build-cmd.sh

```diff
@@ -44,14 +44,25 @@ pushd "$JPEGLIB_SOURCE_DIR"
                 "150")
                     target="setup-v15"
                     ;;
+                "160")
+                    target="setup-v16"
+                    ;;
+                "170")
+                    target="setup-v17"
+                    ;;
```

Comment:
Really tempting to change the `*)` stanza to set

    target="setup-v${AUTOBUILD_VSVER:0:2}"

echo a message explaining that it's unrecognized but this is what we're assuming, and carry on.